packages/phoenix-event-display/src/tests/managers/three-manager/effects-manager.test.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * @jest-environment jsdom
+ */
+import {
+  Camera,
+  Scene,
+  WebGLRenderer,
+  Color,
+  WebGLRenderTarget,
+  Texture,
+  MeshDepthMaterial,
+  ShaderMaterial,
+} from 'three';
+import { OutlinePass } from 'three/examples/jsm/postprocessing/OutlinePass';
+import { FullScreenQuad } from 'three/examples/jsm/postprocessing/Pass';
+import { RenderPass } from 'three/examples/jsm/postprocessing/RenderPass';
+import { EffectsManager } from '../../../managers/three-manager/effects-manager';
+import createRenderer from '../../helpers/create-renderer';
+
+describe('EffectsManager', () => {
+  let effectsManager: EffectsManager;
+  let camera: Camera;
+  let scene: Scene;
+  let renderer: WebGLRenderer;
+
+  beforeEach(() => {
+    camera = new Camera();
+    scene = new Scene();
+    renderer = createRenderer();
+    effectsManager = new EffectsManager(camera, scene, renderer);
+  });
+
+  it('should create an instance', () => {
+    expect(effectsManager).toBeTruthy();
+  });
+
+  it('should initialize the outline pass for highlighting hovered over event display elements', () => {
+    const outlinePass = effectsManager.addOutlinePassForSelection();
+
+    expect(outlinePass).toBeInstanceOf(OutlinePass);
+    expect(outlinePass.selectedObjects.length).toBe(0);
```

Comment:
I don't think there is a need to check all these properties. We just need to assert if the composer contains the outline pass after we add it to the composer.